test/recipes/80-test_cmp_http.t

```diff
@@ -278,19 +276,25 @@ sub start_mock_server {
     my $pid = open($server_fh, "$cmd|") or die "Trying to $cmd";
     print "Pid is: $pid\n";
     if ($server_port eq "0") {
+        my $accept_msg;
+
         # Find out the actual server port
         while (<$server_fh>) {
             print;
             s/\R$//;                # Better chomp
             next unless (/^ACCEPT/);
```

Comment:
This still loops until ACCEPT is found, which as I warned will cause hangs if the server misbehaves.

Reply:
Yes.  We have discussed this to death.  I think we should treat that separately, *if* we get into that situation (again)